src/azure-cli-core/azure/cli/core/auth/identity.py

```diff
@@ -29,6 +29,9 @@
 AZURE_CLIENT_ID = "AZURE_CLIENT_ID"
 AZURE_CLIENT_SECRET = "AZURE_CLIENT_SECRET"
 
+WAM_PROMPT = (
+    "Please select the account you want to log in with. "
+    "For more information on login with Azure CLI, see https://go.microsoft.com/fwlink/?linkid=2271136")
```

Comment:
Our message we had in the Teams group chat was

```suggestion
    "For more information about login with Azure CLI go to https://go.microsoft.com/fwlink/?linkid=2271136")
```

I reworded it a little bit to be consistent with other messages.